types/zfont/index.d.ts

```diff
@@ -0,0 +1,226 @@
+// Type definitions for zfont 1.2
+// Project: https://github.com/jaames/zfont
+// Definitions by: Dmitry Demensky <https://github.com/demensky>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.4
+
+import * as zdog from 'zdog';
+
+export as namespace Zfont;
+
+export function init(x: typeof zdog): typeof zdog;
+
+declare module 'zdog' {
```

Comment:
Instead of declaring this as ambient module here please add zdog as another package and move definitions there. 
Thank you.

Reply:
If `zdog` package already present you just need to reference it instead of redefining it

Reply:
@sheetalkamat. `zdog` is a separate library (I also wrote the definitions for it).
 I also do not understand why the author (@jaames) simply did not export entities ([usage example](https://github.com/jaames/zfont#hello-world)).

@sheetalkamat please provide an example of how best to do this.

Reply:
please take a look at https://github.com/DefinitelyTyped/DefinitelyTyped#im-writing-a-definition-that-depends-on-another-definition-should-i-use-reference-types--or-an-import 
If something is missing or wrong in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/zdog/index.d.ts you can fix it instead and please remove the definition of zdog from here.

Reply:
@sheetalkamat [`zdog`](https://www.npmjs.com/package/zdog) & [`zfont`](https://www.npmjs.com/package/zfont) are different packages. `@types/zdog` is implemented correctly.

`zfont` patch the `zdog` API.

Reply:
Got it .. I didn't realise that it was augmentation instead of definition. Thanks